### PR TITLE
added "JavaScript" project

### PR DESCRIPTION
**Pull request by kshitizsaini113**

### CODE REVIEW

File: _data/projects/JavaScript.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: JavaScript
+desc: Repository containing various JavaScript cose
```
Comment (shiftkey) [inline _data/projects/JavaScript.yml]: @kshitizsaini113 can we be a bit more descriptive here about what the project is for? It's not clear to me from the repository itself how contributors can help out.

Comment (kshitizsaini113) [reply _data/projects/JavaScript.yml]: @shiftkey I have updated the description. You may have a look at it.

**Comment by shiftkey** [comment]
@kshitizsaini113 changes look fine, but there are no open issues on the repository currently so it may not be listed on the site immediately. Let me know if you intend to add issues that other contributors can help out with.

**Review by shiftkey — APPROVED** [review]

File: _data/projects/JavaScript.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: JavaScript
```
Comment (ritwik12) [inline _data/projects/JavaScript.yml]: @shiftkey @kshitizsaini113 Wouldn't it be better to use some other name than just JavaScript? I don't want users to get Confused with the original JavaScript programming language repository.

Comment (shiftkey) [reply _data/projects/JavaScript.yml]: Great point @ritwik12. 

@kshitizsaini113 is there a better name here to describe the project?

Comment (kshitizsaini113) [reply _data/projects/JavaScript.yml]: I have changed the name

**Review by shiftkey — APPROVED** [review]

File: _data/projects/JavaScript.yml (hunk 2 above)
Comment (ritwik12) [reply _data/projects/JavaScript.yml]: Thanks

**Pull request merged**